src/components/Post/Post.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
```

Comment:
I think this file should be in the views folder. Everything rendered by the Router in App.js are views (except for the homepage which is written inline.